authors/apps/profiles/serializers.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import serializers
+from .models import UserProfile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    """
+    Class to serialize user profile data
+    """
+    profile_image = serializers.ImageField(default='No image')
+    gender = serializers.RegexField(
+        regex='^(N|F|M)$',
+        write_only=True,
+        error_messages={
+            'required': 'Password field required',
```

Comment:
What is Password field doing in genders?

Reply:
FIxed it